src/PrestaShopBundle/Controller/Admin/Configure/AdvancedParameters/PerformanceController.php

```diff
@@ -64,6 +65,20 @@ public function indexAction(FormInterface $form = null)
 
         $form = is_null($form) ? $this->get('prestashop.adapter.performance.form_handler')->getForm() : $form;
 
+        if (!in_array(
+            $this->authorizationLevel($this::CONTROLLER_NAME),
+            array(
+                PageVoter::LEVEL_READ,
+                PageVoter::LEVEL_UPDATE,
+                PageVoter::LEVEL_CREATE,
+                PageVoter::LEVEL_DELETE,
```

Comment:
we've been doing this kind of checks, but is it correct? in BO you can assign user to have only `create` permissions, does it mean user can `read` as well? :thinking:

Reply:
thinking about it, we should not at this check in the core at all, only the one about clear cache action.